### PR TITLE
Remove AUheadings to AUheading

### PR DESCRIPTION
**Pull request by Reshma-M-A**
Tobias noticed there was an inconsistency in the code that we reference `AUHeadings` when it should be `AUHeading`. 

So this PR will fix that issue.

### CODE REVIEW

File: apps/marketplace/main.scss [hunk 1]
```diff
@@ -26,6 +26,10 @@
   }
 }
 
+.marginTop0 {
+  margin-top: 0rem
```
Comment (Reshma-M-A) [inline apps/marketplace/main.scss]: Once Mar 3481 is merged this will make sense.

File: apps/marketplace/components/BuyerATM/BuyerATMAdditionalInformationStage.js [hunk 2]
```diff
@@ -2,19 +2,45 @@ import React, { Component } from 'react'
...
+import DateControl from 'marketplace/components/BuyerBriefFlow/DateControl'
+import styles from 'marketplace/main.scss'
 
-class BuyerATMAdditionalInformationStage extends Component {
+const requiredContactNumber = v => required(v.contactNumber)
+const contactNumberFormat = v => validPhoneNumber(v.contactNumber)
+const requiredClosedAt = v => required(v.closedAt)
+const closedAtIsValid = v => validDate(v.closedAt)
+const closedAtIs2DaysInFuture = v => !closedAtIsValid(v) || dateIs2DaysInFuture(v.closedAt)
+const closedAtIsBefore = v => !closedAtIsValid(v) || dateIsBefore(v.closedAt, addDays(new Date(), 364))
```
Comment (tobiasbrown) [inline apps/marketplace/components/BuyerATM/BuyerATMAdditionalInformationStage.js]: Why are these changes here?  They look like they might be from another one of your pull requests.

File: apps/marketplace/components/BuyerATM/BuyerATMAdditionalInformationStage.scss [hunk 3]
```diff
@@ -1,3 +0,0 @@
-.closingDateControl {
-  margin-top: 1.5rem;
```
Comment (tobiasbrown) [inline apps/marketplace/components/BuyerATM/BuyerATMAdditionalInformationStage.scss]: Should this change be in this pull request?

File: apps/marketplace/components/BuyerATM/BuyerATMStages.js [hunk 4]
```diff
@@ -74,8 +73,7 @@ const BuyerATMStages = [
     slug: 'additional',
     title: 'Additional information',
     component: BuyerATMAdditionalInformationStage,
-    isDone: formValues =>
-      dateIs2DaysInFuture(formValues.closedAt) && formValues.contactNumber && validPhoneNumber(formValues.contactNumber)
+    isDone: additionalDone
```
Comment (tobiasbrown) [inline apps/marketplace/components/BuyerATM/BuyerATMStages.js]: Should these changes be in this pull request?

File: apps/marketplace/components/BuyerSpecialist/BuyerSpecialistAdditionalInformationStage.js [hunk 5]
```diff
@@ -117,12 +117,12 @@ export class BuyerSpecialistRequirementsStage extends Component {
             closedAtIsValid: 'You must enter a valid closing date',
             closedAtIs2DaysInFuture: 'You must enter a closing date at least 2 days from now',
             requiredClosedAt: 'You must enter the closing date for this opportunity',
-            closedAtIsBefore: 'You must enter a closing date not more than one year from now'
+            closedAtIsBefore: 'You must enter a closing date no more than one year from now'
```
Comment (tobiasbrown) [inline apps/marketplace/components/BuyerSpecialist/BuyerSpecialistAdditionalInformationStage.js]: Should this change be in this pull request?

File: apps/marketplace/main.scss (hunk 1 above)
Comment (tobiasbrown) [reply apps/marketplace/main.scss]: Should these changes be added with the pull request for MAR-3481?
